specification/compute/resource-manager/Microsoft.Compute/stable/2018-06-01/compute.json

```diff
@@ -7067,6 +7067,13 @@
           "readOnly": true,
           "type": "string",
           "description": "The provisioning state, which only appears in the response."
+        },
+        "provisionAfterExtensions": {
```

Comment:
Out of curiosity, what is the service behavior if 
1) a VMSS is PUT with provisionAfterExtension, as specified in this version,
2) a lower-version client does a GET & PUT without provisionAfterExtension specified
Is the property wiped out, is the PUT failed, or is the prior value preserved(merge)? Does the behavior cause undesired effects?